boards/arm/relax_kit/doc/index.rst

```diff
@@ -0,0 +1,82 @@
+.. _relax_kit:
+
+INFINEON RELAX-KIT
+##################
+
+Overview
+********
+
+TheXMC4500 Relax Kit designed to evaluate the capabilities of the XMC4500
+Microcontroller. It contains High performance ARM Cortex-M4F which can run
```

Comment:
nit: instead of "contains", "includes" or "is based on"

Reply:
Fixed, thanks.